configuration/wireless/headless.md

```diff
@@ -8,13 +8,16 @@ Once an image is created on an SD card, by inserting it into a card reader on a
 
 You will need to define a `wpa_supplicant.conf` file for your particular wireless network. Put this file in the boot folder, and when the Pi first boots, it will copy that file into the correct location in the Linux root file system and use those settings to start up wireless networking.
 
+You can use notepad++ so that the file is saved in correct formats. You can go in edit--> EOL Conversion--> UNIX so that Raspbian understands the code properly.Paste the below After that and save it.
+
 `wpa_supplicant.conf` file example:
```

Comment:
I think this need to be more generic since the user could be using Linux to edit the file, or not have Notepad++, perhaps something along the lines of "Make sure you have Unix line endings enabled".